letsencrypt/models/letsencrypt.py

```diff
@@ -149,7 +158,11 @@ def cron(self):
             crt.write(chain_cert.read())
             chain_cert.close()
             _logger.info('wrote %s', crt.name)
-        self.call_cmdline([
-            'sh', '-c', self.env['ir.config_parameter'].get_param(
-                'letsencrypt.reload_command', ''),
-        ])
+        reload_cmd = self.env['ir.config_parameter'].get_param(
+            'letsencrypt.reload_command', False)
+        if reload_cmd:
+            _logger.info('reloading webserver...')
+            self.call_cmdline(['sh', '-c', reload_cmd])
```

Comment:
if you need multiple commands, you can write `sh -c 'you-command'` in the config parameter already. And if you follow https://github.com/OCA/server-tools/pull/347/files#r63487176, you can write those multiple commands in the extra script, no matter the OS.